udata/rdf.py

```diff
@@ -0,0 +1,245 @@
+
+
+# JSON-LD context used for udata DCAT representation
+CONTEXT = {
+    # Namespaces
+    '@vocab': 'http://www.w3.org/ns/dcat#',
+    'dcat': 'http://www.w3.org/ns/dcat#',
+    'dct': 'http://purl.org/dc/terms/',
+    'foaf': 'http://xmlns.com/foaf/0.1/',
+    'org': 'http://www.w3.org/ns/org#',
+    'rdfs': 'http://www.w3.org/2000/01/rdf-schema#',
+    'skos': 'http://www.w3.org/2004/02/skos/core#',
+    'spdx': 'http://spdx.org/rdf/terms#',
+    'vcard': 'http://www.w3.org/2006/vcard/ns#',
+    'schema': 'http://schema.org/',
+    'hydra': 'http://www.w3.org/ns/hydra/core#',
+    'freq': 'http://purl.org/cld/freq/',
+    # Aliased field
+    'downloadURL': {
+        '@id': 'dcat:downloadURL',
+        '@type': '@id'
+    },
+    'accessURL': {
+        '@id': 'dcat:accessURL',
+        '@type': '@id'
+    },
+    'dataset': {
+        '@id': 'dcat:dataset',
+        '@type': '@id'
+    },
+    'distribution': {
+        '@id': 'dcat:distribution',
+        '@type': '@id'
+    },
+    'title': 'dct:title',
+    'description': 'dct:description',
+    'issued': {
+        '@id': 'dct:issued',
+        '@type': 'http://www.w3.org/2001/XMLSchema#dateTime'
+    },
+    'modified': {
+        '@id': 'dct:modified',
+        '@type': 'http://www.w3.org/2001/XMLSchema#dateTime'
+    },
+    'language': 'dct:language',
+    'license': 'dct:license',
+    'rights': 'dct:rights',
+    'spatial': 'dct:spatial',
+    'identifier': 'dct:identifier',
+    'temporal': 'dct:temporal',
+    'format': 'dct:format',
+    'accrualPeriodicity': 'dct:accrualPeriodicity',
+    'homepage': {
+        '@id': 'foaf:homepage',
+        '@type': '@id'
+    },
+    'publisher': {
+        '@id': 'dct:publisher',
+        '@type': '@id'
+    },
+    'fn': 'vcard:fn',
+    'hasEmail': 'vcard:email',
+    'subOrganizationOf': 'org:subOrganizationOf',
+    'checksum': 'spdx:checksum',
+    'algorithm': {
+        '@id': 'spdx:algorithm',
+        '@type': '@id'
+    },
+    'checksumValue': 'spdx:checksumValue',
+    'label': 'rdfs:label',
+    'name': 'foaf:name',
+    'startDate': 'schema:startDate',
+    'endDate': 'schema:endDate',
+    'view': {
+        '@id': 'hydra:view',
+        '@type': '@id'
+    },
+    'first': {
+        '@id': 'hydra:first',
+        '@type': '@id'
+    },
+    'last': {
+        '@id': 'hydra:last',
+        '@type': '@id'
+    },
+    'next': {
+        '@id': 'hydra:next',
+        '@type': '@id'
+    },
+    'previous': {
+        '@id': 'hydra:previous',
+        '@type': '@id'
+    },
+    'totalItems': 'hydra:totalItems',
+}
+
+
+class UDataContext(Context):
+    '''
+    An hackish way to serialize context as a root relative URL.
+
+    Exploit this issue https://github.com/RDFLib/rdflib-jsonld/issues/37
+    and the fact that this method is used to render the context in the
+    resulting JSON-LD.
+
+    See: https://github.com/RDFLib/rdflib-jsonld/blob/master/rdflib_jsonld/serializer.py#L101-L103
+    '''  # noqa
```

Comment:
If you put only a URL on one line, the linter shouldn't complain even without a # noqa comment. At least mine 😉